PrincessBrideTrivia/PrincessBrideTrivia/Question.cs

```diff
@@ -1,9 +1,16 @@
-﻿namespace PrincessBrideTrivia
+namespace PrincessBrideTrivia
 {
     public class Question
     {
         public string Text;
         public string[] Answers;
         public string CorrectAnswerIndex;
+
+        public Question(string Q, string[] A, string a)
+        {
+            this.Text = Q;
+            this.Answers = A;
+            this.CorrectAnswerIndex = a;
+        }
```

Comment:
```suggestion
        public Question(string Q, string[] A, string a)
        {
            Text = Q;
            Answers = A;
            CorrectAnswerIndex = a;
        }
```
You do not need to use this `this` key word here.

Additionally, your code does not compile as you did not update your test methods to implement this new constructor.